addons/crm_livechat/models/chatbot_script_step.py

```diff
@@ -62,4 +70,19 @@ def _process_step_create_lead(self, discuss_channel):
         create_values.update(self._chatbot_crm_prepare_lead_values(
             discuss_channel, customer_values['description']))
 
-        self.env['crm.lead'].create(create_values)
+        return self.env['crm.lead'].create(create_values)
+
+    def _process_step_create_lead_and_forward(self, discuss_channel, posted_message):
+        lead = self._process_step_create_lead(discuss_channel)
+        team = lead.team_id
+        team_operators = self.env["res.users"]
+        if team and lead:
+            team_operators = team.crm_team_member_ids.filtered(lambda m: m._get_assignment_quota() > 0).user_id
+        posted_message = self._process_step_forward_operator(discuss_channel.with_context(crm_team_id=lead.team_id), posted_message, team_operators)
```

Comment:
You would need to rebase and take into account the new conditions in `_compute_available_operator_ids`.

In particular, if the channel is marked as limited, ignore users having more sessions than `max_sessions_mode` or having a call in progress.